models/User.js

```diff
@@ -1,15 +1,27 @@
 var keystone = require('@twreporter/keystone');
 var transform = require('model-transform');
 var Types = keystone.Field.Types;
+var config = require('../config');
 
 var User = new keystone.List('User');
 
+const bucket = config['options']['gcs config']['bucket']['resizedImage'];
```

Comment:
I supposed we should use different bucket?

Reply:
Yes, I second that.

How about creating a new bucket 'twreporter-avatars' for production usage, what do you say ?

And this line will be:
`bucket = config['options']['gcs config']['bucket']['avatar']`

Reply:
I suppose we could just put in `twreporter-multimedia`?

Reply:
Updated in https://github.com/twreporter/plate/pull/263/commits/0366f0d53fcf3fec0228e985313b7e2ef5a15595 and https://github.com/twreporter/kubernetes-config/pull/40.